apps/antalmanac/src/App.tsx

```diff
@@ -45,9 +46,11 @@ export default function App() {
         <AppQueryProvider>
             <AppThemeProvider>
                 <AppThemev5Provider>
-                    <SnackbarProvider>
-                        <RouterProvider router={BrowserRouter} />
-                    </SnackbarProvider>
+                    <Show24HourTimeProvider>
+                        <SnackbarProvider>
+                            <RouterProvider router={BrowserRouter} />
+                        </SnackbarProvider>
+                    </Show24HourTimeProvider>
```

Comment:
Don't use context for this.

```suggestion
                        <SnackbarProvider>
                            <RouterProvider router={BrowserRouter} />
                        </SnackbarProvider>
```